Updates/Instances/532_karazhan.sql

```diff
@@ -3643,6 +3643,16 @@ INSERT INTO `dbscripts_on_creature_movement` (`id`, `delay`, `command`, `datalon
 -- npcs are Spectral Servant, Skeletal Waiter and Bennetthey ill prob get dont via creature_ai_script
 DELETE FROM dbscripts_on_creature_movement WHERE id IN (1640701, 1641501, 1641502, 1642601, 1642607, 1642611);
 
+-- 2 Ghostly stewards RP script on The Scullery
+DELETE FROM dbscripts_on_relay WHERE id = 1641401;
+REPLACE INTO `dbscripts_on_relay` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `buddy_entry`, `search_radius`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `comments`) VALUES
+(1641401, 0, 0, 31, 16414, 3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Karazhan - Ghostly Steward - Terminate Script if 2nd Ghostly Steward Found'), -- this is probably wrong, cause the npc still does the rp even if the friend is dead, but this would cause db errors 
```

Comment:
https://youtu.be/brVWEoAy8xg 
Just to let this here, retail pov (but same happend on wotlk classic)
He is still able to do this rp even if the other npc is dead. I still want to let this part in to prevent dberrors.